src/main/java/com/parsing/parsers/parserlot/Constants.java

```diff
@@ -0,0 +1,49 @@
+package com.parsing.parsers.parserlot;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Constants {
+
+    public final static LocalDate START_DATE = LocalDate.of(2023, 02, 21);
+    public final static LocalDate END_DATE = LocalDate.of(2023, 02, 21);
+    private static final int START_LOT = 0;
+    private static final int END_LOT = 500;
+
+    public List<String> createCounter() {
```

Comment:
What does counter mean? Not sure if I really understand what this method does with this name

Reply:
And why is it in Constants?

Reply:
I have changed to URLListBuilder. It has a more good name, but I think that exists better